tutorial/tutorial_chat_module_getting_started.ipynb

```diff
@@ -0,0 +1,473 @@
+{
```

Comment:
let us move to mlc-llm folder, so it is clear it is tutorial for mlc-llm

Reply:
@tqchen do you mean `notebooks/tutorial/mlc-llm` or `notebooks/mlc-llm`?

Reply:
notebooks/mlc-llm is fine

Reply:
Addressed in https://github.com/mlc-ai/notebooks/pull/12/commits/69a07702edebd5371fa3483782e489c0952783cc.